docs/operation/ros_interface/ros1.rst

```diff
@@ -80,12 +80,55 @@ The **slate_base_node** has the following parameters:
   * **Description**: The name of the base_link frame when publishing the odom->base_link TF.
   * **Default**: ``"base_link"``
 
-Usage
------
+Examples
+--------
+
+The **slate_base_node** is designed to be controlled by other user-written nodes that publish to the ``cmd_vel`` topic or call the services provided by the **slate_base_node**. Below are some examples of how to interact with the **slate_base_node** using ROS 1 command line tools.
+
+Setting Parameters
+^^^^^^^^^^^^^^^^^^
+
+You can set parameters for the **slate_base_node** when running it using the ``_param`` flag.
+
+.. code-block:: bash
+
+  # Enable publishing of the odom->base_link TF
+  $ rosrun interbotix_slate_driver slate_base_node _publish_tf:=true
+
+  # Set the odometry frame name to "my_odom"
+  $ rosrun interbotix_slate_driver slate_base_node _odom_frame_name:="my_odom"
+
+  # Set the base frame name to "my_base"
+  $ rosrun interbotix_slate_driver slate_base_node _base_frame_name:="my_base"
+
+Calling Services
+^^^^^^^^^^^^^^^^
 
-Using rosrun
-^^^^^^^^^^^^
+You can call services provided by the **slate_base_node** using the ``rosservice call`` command.
 
 .. code-block:: bash
 
-  $ rosrun interbotix_slate_driver slate_base_node
+  # Disable charging
+  $ rosservice call /enable_charging std_srvs/SetBool "{data: false}"
+
+  # Set the lights to color BLUE
+  $ rosservice call /set_light_state interbotix_slate_msgs/SetLightState "{light_state: 4}"
+
+  # Set the text to "hello world" on the SLATE screen
+  $ rosservice call /set_text interbotix_slate_msgs/SetString "{data: 'hello world'}"
+
+  # Disable torque on the drive wheels
+  $ rosservice call /set_motor_torque_status std_srvs/SetBool "{data: false}"
+
+Publishing Velocity Commands
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can publish velocity commands to the **slate_base_node** using the ``rostopic pub`` command. ``-r 10`` sets the publishing rate to 10 Hz.
```

Comment:
Add a warning admonition here that the robot will move and a note that they can press :kbd:`Ctrl` + :kbd:`C` to stop the robot at any time.